src/harness/fourslash.ts

```diff
@@ -2736,6 +2749,26 @@ namespace FourSlash {
             }
         }
 
+        public verifyRefactorAvailable(negative: boolean, name?: string, subName?: string) {

```

Comment:
>negative [](start = 39, length = 8)

We seem to have an existing pattern, but this is a particularly pointed example of using a predicate with a negative name.